File: Beecrowd/Beginner/2203 - ~~/2344 - Notas da Prova/Python/2344_NotasdaProva.py
grade = int(input())

if grade == 0:
    print("E")
elif 1 <= grade <= 35:
    print("D")
elif 36 <= grade <= 60:
    print("C")
elif 61 <= grade <= 85:
    print("B")
elif 86 <= grade <= 100:
    print("A")

File: Beecrowd/Beginner/2203 - ~~/2344 - Notas da Prova/C++/2344_NotasdaProva.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int grade;
    cin >> grade;

    if (grade == 0)
        cout << "E" << endl;
    else if (grade >= 1 && grade <= 35)
        cout << "D" << endl;
    else if (grade >= 36 && grade <= 60)
        cout << "C" << endl;
    else if (grade >= 61 && grade <= 85)
        cout << "B" << endl;
    else if (grade >= 86 && grade <= 100)
        cout << "A" << endl;

    return 0;
}